uber/models.py

```diff
@@ -539,6 +539,8 @@ def get_next_badge_num(self, badge_type):
                         and attendee.badge_num <= self.auto_badge_num(badge_type):
                     new_badge_num = max(self.auto_badge_num(badge_type), 1 + attendee.badge_num)
 
+            assert new_badge_num < c.BADGE_RANGES[badge_type][1], 'There are no more badge numbers available in this range!'
```

Comment:
I added this here instead of in auto_badge_num because it's more useful to check the badge number after we're done pushing it upwards to accommodate dirty sessions.